source/developer/mobile-developer-setup.rst

```diff
@@ -117,10 +189,122 @@ Android (Device)
 
 8. The installed APK may not be opened automatically. You may need to manually open the Mattermost app on your device.
 
+Start Developing and Building
+------------------------------------
+
+In order to develop and build the Mattermost mobile apps you'll need to get a copy of the source code. Also, it is best if you fork our mattermost-mobile repo since it is most likely that you'll want to contribute back your work.
+
+1. Fork the mattermost-mobile repository on GitHub.
+2. Clone your fork locally:
+ - Open a terminal 
+ - Change to a directory you want to hold your local copy 
+ - Run ``git clone https://github.com/<username>/mattermost-mobile.git`` if you want to use HTTPS, or ``git clone git@github.com:<username>/mattermost-mobile.git`` if you want to use SSH
+
+Important: <username> refers to the username or organization in GitHub that forked the repository.
+
+3. Change the directory to ``mattermost-mobile``.
+ - ``cd mattermost-mobile``
+
+4. Run ``make pre-run`` in order to install all the dependencies.
+ - It is highly important that you run everything with the make commands and avoid using npm or yarn to install dependencies.
+ 
+Make Commands Explained
+------------------------------------
+
+We've included a bunch of make commands in order to control the development flow and to ensure that everything works as expected. Always try and use these make commands unless what you trying to do can't be accomplished by one of these commands.
+
+Every make command has to be run from a terminal in the project's root directory.
+
+Commands to Prepare the App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These make commands are used to install dependencies, to configure necessary steps before running or building the app, and to clean everything.
+
+ - **make pre-run**: Downloads any project dependencies needed and will set up the app assets. This is the equivalent of running the .yarninstall, .podinstall and dist/assets make commands.
+ - **make .yarninstall**: Downloads JavaScript and react native dependencies. Once this command finishes executing it will create a `.yarninstall` empty file in the project's root directory to indicate that this command already ran. If yarn updates your local dependencies in the `node_modules` directory, it will automatically run `make post-install` for you.
```

Comment:
I think this needs two ` for code formatting in rst.

Can we also change the .yarninstall, .podinstall, dist/assets in the previous point to have the same formatting